dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_item.tsx

```diff
@@ -228,22 +249,53 @@ export const DataConfigPanelItem = ({ fieldOptionList, visualizations }: any) =>
       </>
     ));
 
+  const getNumberField = (type: string) => (
+    <>
+      <EuiFieldNumber
+        id={htmlIdGenerator('input-number')()}
+        fullWidth
+        placeholder="auto"
+        value={configList.dimensions[0][type] ? configList.dimensions[0][type] : ''}
+        onChange={(e) => updateHistogramConfig('dimensions', type, e.target.value)}
+        data-test-subj="valueFieldNumber"
+      />
+      <EuiSpacer size="s" />
+    </>
+  );
+
   return (
     <>
       <EuiTitle size="xxs">
         <h3>Data Configurations</h3>
       </EuiTitle>
       <EuiSpacer size="s" />
-      <EuiTitle size="xxs">
-        <h3>Dimensions</h3>
-      </EuiTitle>
-      {getCommonUI(configList.dimensions, 'dimensions')}
+      {visualizations.vis.name !== visChartTypes.Histogram ? (
+        <>
+          <EuiTitle size="xxs">
+            <h3>Dimensions</h3>
+          </EuiTitle>
+          {getCommonUI(configList.dimensions, 'dimensions')}
```

Comment:
check formatting of complete code.

Reply:
This file shows no changes in formatting after selecting the whole document and clicking on format document or by doing Alt+Shift+F, so no formatting is needed.